Extending/No-Published-Content/index.md

```diff
@@ -0,0 +1,9 @@
+# No Published Content Page
+
+When Umbraco is first installed, if no starter kit is chosen, there will be no content created in the backoffice, and no published content to display.
+
+Content can also be created, but be unpublished.
+
+If browsing the front-end of the website, when this situation of no published content is found, a custom page is displayed.
+
+This page is served from `/config/splashes/noNodes.aspx`.
```

Comment:
```suggestion
This page is served from `/config/splashes/NoNodes.cshtml`.
```

Reply:
Actually, I think this is right as is.  `Index.md` is for the current version, i.e. v8, which does use the `.aspx` page.  The .Net Core version of the documentation page has the extension `-v9.md`